code/modules/countdown/countdown.dm

```diff
@@ -51,7 +51,7 @@
 	displayed_text = new_val
 
 	if(displayed_text)
-		maptext = "<font size = [text_size]>[displayed_text]</font>"
+		maptext = "<font face='Small Fonts' size=[text_size]>[displayed_text]</font>"
```

Comment:
Oh thank god 

For context, this makes text look crisp (See brigtimers) and not an anti-aliased mess like it currently is.